<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Winamp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/ico/ico/preferences_desktop_sound_16925.ico</normaloff>:/ico/ico/preferences_desktop_sound_16925.ico</iconset>
  </property>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelComposition">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel Composition</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrev">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderVolume">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolume">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume:</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderProgress">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>731</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelProgress">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Progress</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDuration">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMute">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewPlaylist">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>171</y>
     <width>721</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLoop">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>560</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Loop</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxShuffle">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>560</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCLR">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>560</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CLR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDEL">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>560</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DEL</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
